Gems/MotionMatching/Code/Source/Feature.h

```diff
@@ -76,6 +78,7 @@ namespace EMotionFX::MotionMatching
 
             size_t m_frameIndex = InvalidIndex;
             const Pose* m_framePose = nullptr; //! Pre-sampled pose for the given frame.
```

Comment:
suggestion: Is it worth dropping `const` here as opposed to casting away `const` in `ExtractFeatureValuesRange`?